src/aspire/utils/coor_trans.py

```diff
@@ -153,6 +153,17 @@ def uniform_random_angles(n, seed=None, dtype=np.float32):
             np.arccos(2 * np.random.random(n) - 1),
             np.random.random(n) * 2 * np.pi
         ))
+
+    # TODO, If/when unit tests refactoring allows, we should swap out
+    #   for internal random calls. This will facility moving from
+    #   RandomState to Generator.
+    # angles = np.column_stack((
```

Comment:
It's not clear to me what this code is doing here. Ideally, we don't want to be calling `random` several times over with the same seed (i.e., what we're doing above seems more appropriate). Perhaps I'm missing something here.

Reply:
See the longer note below. I think by default our Random tools should effectively manage the situation in a way that those two invocations would be equivalent... and would prefer just use our wrappers to manage the contexts than developers coding the contexts.

IME it is actually important to iterate the RNG in practice.  What we do now (fine for UT I guess, maybe...) resets RNG virtually all the time.  My stance is that we only want to reset under very specific and explicit circumstances.

Reply:
I removed this comment for now, citing this discussion. I do think there is still a lot of work todo wrt Random, but I was mainly just trying to get a little bit of work done in that direction between other larger (or blocked) tasks.